tox.ini

```diff
@@ -95,3 +95,29 @@ setenv =
 commands =
     pip install -e .
 	behave --tags=-disabled integration_tests
+
+
+[testenv:ci_gsye_122]
+basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH
+deps = pip-tools
+whitelist_externals = git
+                      /bin/rm
+                      /bin/cp
+                      /bin/mkdir
+                      echo
+					  bash
+commands =
+		python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+		pip-sync requirements/tests.txt requirements/pandapower.txt
+		pip install -e .
+		pip uninstall -y gsy-framework
+		pip install git+https://github.com/gridsingularity/gsy-framework@{env:BRANCH:master}
+		rm -rf gsy-backend-integration-tests
+		rm -rf integration_tests
+		mkdir integration_tests
+		git clone --branch {env:BRANCH:master} git@github.com:gridsingularity/gsy-backend-integration-tests.git
+		cp -r ./gsy-backend-integration-tests/gsy_e_tests/integration_tests/ ./integration_tests/
+		bash -c 'for f in ./integration_tests/*.feature; do behave --tags=-slow --tags=-disabled "$f" || exit 1; done'
```

Comment:
How about:
```suggestion
		git clone --branch {env:BRANCH:master} git@github.com:gridsingularity/gsy-backend-integration-tests.git 
		bash -c 'for f in ./gsy-backend-integration-tests/gsy_e_tests/*.feature; do behave --tags=-slow --tags=-disabled "$f" || exit 1; done'
```

Reply:
On the first try,  I did something similar to what you suggest. However the references to other modules on the step files and the path to other files were affected and for it to work they needed to be edited. The difference though, was that the gsy-e project was not held on the root of the project, but on a subfolder, causing the problem mentioned above.
This time, gsy-e it is indeed on the root and therefore what you suggest works, so I'll change it. (I discovered the reason why after trying it out again)